src/widgets/category/title/search-title-view.js

```diff
@@ -54,6 +55,9 @@ module.exports = cdb.core.View.extend({
     this.add_related_model(this.model.lockedCategories);
     this.dataviewModel.filter.bind('change', this.render, this);
     this.add_related_model(this.dataviewModel.filter);
+
+    this.dataviewModel.layer.bind('change:visible', this.render, this);
```

Comment:
Really weird to have a bind within a render function, because each time render function is fired, a new bind is set...